src/client.py

```diff
@@ -25,6 +27,10 @@ def __init__(self, email: str, password: str, box_id: int, box_name: str):
         self.box_id = box_id
         self.box_name = box_name
 
+        self.logger = logging.getLogger(__name__)
```

Comment:
what's the point of logging in this simple project? I mean if there's something wrong the exceptions right now are pretty self explanatory

Reply:
That's because I like to have more info in plain English to identify issues quicker on the console :)

Reply:
the exception we are using here can have a message on it, so I don't understand what more you need